scribejava-httpclient-okhttp/src/main/java/com/github/scribejava/httpclient/okhttp/OkHttpHttpClient.java

```diff
@@ -191,9 +191,10 @@ static Response convertResponse(okhttp3.Response okHttpResponse) {
             headersMap.put(headerName, headers.get(headerName));
         }
 
-        final ResponseBody body = okHttpResponse.body();
-        return new Response(okHttpResponse.code(), okHttpResponse.message(), headersMap,
-                body == null ? null : body.byteStream());
+        try (ResponseBody body = okHttpResponse.body()) {
+            return new Response(okHttpResponse.code(), okHttpResponse.message(), headersMap,
+                    body == null ? null : body.byteStream());
+        }
     }
```

Comment:
shouldn't there be a finally? i don't get how this closes the body.

Reply:
This is a try-with-resources. https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
ok cool

Reply:
Hi, @drei01 , ResponseBody shouldn't be closed here in such way.
it is passed to the com.github.scribejava.core.model.Response.
and is used by it to read body stream later.
Developers should close com.github.scribejava.core.model.Response. And it will close the stream.
Response implements Closeable and have method
```
    @Override
    public void close() throws IOException {
        if (stream != null) {
            stream.close();
        }
    }
```

Reply:
@kullfar thanks for the clarification. I'll close this PR.